sdk/OoyalaSkinSDK/StartScreen.js

```diff
@@ -131,19 +168,20 @@ var StartScreen = React.createClass({
 
   render: function() {  
     var promoImage = this.getPromoImage();
-    var playButton = this.getPlayButton();
+    var PlayButton = this.getPlayButton();
     var infoPanel = this.getInfoPanel();
     var waterMarkImage = this.getWaterMark();
-
-    return (
-      <View style={styles.container}>
-        {promoImage}
-        {waterMarkImage}
-        {infoPanel}
-        {playButton}
+ 
+
+return (
+     <View style={styles.container}>
+       {promoImage}
+       {waterMarkImage}
+       {infoPanel} 
+      {PlayButton}
```

Comment:
just to make sure this gets fixed - make sure this is re-capitalzied after you fix the render